astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -152,7 +152,7 @@ def gcrs_to_icrs(gcrs_coo, icrs_frame):
     else:
         #compute the distance as just the cartesian sum from moving to the SSB
         #we have to do this because the ERFA functions throw away distance info
-        distance = np.sum((-astrom['eb']*u.au + gcrs_coo.cartesian.xyz.T)**2, -1)**0.5
+        distance = np.sum((astrom['eb']*u.au + gcrs_coo.cartesian.xyz.T)**2, -1)**0.5
```

Comment:
This is getting super picky, but I liked to put in `axis=-1`  -- did a double take on this because I thought it was missing the axis specification.

Also, curious as to why you transpose and use the last axis? Won't this be *a tiny tiny tiny* bit faster if you just do 

    distance = np.sum((astrom['eb']*u.au + gcrs_coo.cartesian.xyz)**2, axis=0)**0.5

?